sources.md

```diff
@@ -0,0 +1,10 @@
+# sources
+I will attach a dataset that I found in the wikidata
+
+[SNIRH data]https://snirh.pt/snirh/_dadosbase/site/simplex.php?OBJINFO=DADOS&FILTRA_BACIA=138&FILTRA_COVER=920123704&FILTRA_SITE=920685260
```

Comment:
Please add a `-` at the beginning to make it a list; also the URL must be in parentheses

Reply:
- [SNIRH data] (https://snirh.pt/snirh/_dadosbase/site/simplex.php?OBJINFO=DADOS&FILTRA_BACIA=138&FILTRA_COVER=920123704&FILTRA_SITE=920685260)
-